run.c

```diff
@@ -267,12 +266,13 @@ void find_next_sync_point_and_drift(int with_fof)
 
         set_global_time(nexttime);
 
-        move_particles(All.Ti_nextoutput);
-
+        /* Do domain decomp and rebuild tree before doing
+         * power spectrum or FoF; if the particles have
+         * moved a long way, performance will be degraded.*/
```

Comment:
I am confused. neutrinos will break and performance will be degraded seem to be two different things.

A related issue is last BlueTides run crashed in PM, where it alerted the number of particles included on the nodes marked with ``DEPEND_ON_LOCAL_MASS`` differs from NumPart.

It may be related to this? I haven't got around to look at it closely.